experimental/traffic-portal/src/app/core/origins/detail/origin-detail.component.html

```diff
@@ -0,0 +1,49 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<mat-card appearance="outlined" class="page-content single-column">
+	<tp-loading *ngIf="!type"></tp-loading>
+	<form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="type">
+		<mat-card-header class="headers-container" *ngIf="!new">
+			<a mat-raised-button color="primary" *ngIf="type.useInTable === 'cachegroup'" [routerLink]="'/core/cache-groups'" [queryParams]="{typeName: type.name}">View Cache Groups</a>
+			<a mat-raised-button color="primary" *ngIf="type.useInTable === 'server'" [routerLink]="'/core/servers'" [queryParams]="{type: type.name}">View Servers</a>
+		</mat-card-header>
+		<mat-card-content class="container">
+			<mat-form-field *ngIf="!new">
+				<mat-label>ID</mat-label>
+				<input matInput type="text" name="id" disabled readonly [defaultValue]="type.id" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Name</mat-label>
+				<input matInput type="text" name="name" required [(ngModel)]="type.name" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Description</mat-label>
+				<input matInput type="text" name="description" required [(ngModel)]="type.description" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Use In Table</mat-label>
+				<input matInput type="text" name="useInTable" disabled readonly [defaultValue]="new ? 'server' : type.useInTable" />
+			</mat-form-field>
+			<mat-form-field *ngIf="!new">
+				<mat-label>Last Updated</mat-label>
+				<input matInput type="text" name="lastUpdated" disabled readonly [defaultValue]="type.lastUpdated" />
+			</mat-form-field>
+		</mat-card-content>
```

Comment:
These fields should reflect properties of an Origin, currently they appear to represent a Type